core/src/main/java/aima/core/search/basic/support/BasicProblem.java

```diff
@@ -2,23 +2,17 @@
 
 import java.util.List;
 
-import aima.core.search.api.ActionsFunction;
-import aima.core.search.api.GoalTestPredicate;
-import aima.core.search.api.NondeterministicProblem;
-import aima.core.search.api.OnlineSearchProblem;
-import aima.core.search.api.Problem;
-import aima.core.search.api.ResultFunction;
-import aima.core.search.api.ResultsFunction;
-import aima.core.search.api.StepCostFunction;
+import aima.core.search.api.*;
```

Comment:
Fixed the star imports. @globalworming, 
Actually Alt+Enter of IntelliJ does that. I will take care henceforth.